bot.go

```diff
@@ -59,7 +59,12 @@ func archiveChannels(api *slack.Slack, c []slack.Channel, reason string) {
 		go func(c slack.Channel) {
 			defer wg.Done()
 			if err := api.ArchiveChannel(c.Id); err != nil {
-				log.Printf("Error archiving #%s (%s): %s\n", c.Name, c.Id, err)
+				message := fmt.Sprintf("Error archiving channel #%s (%s): %s\n", c.Name, c.Id, err)
+				log.Printf(message)
+				// send error message in a DM to onErrorNotify user/channel
+				onErrorNotify := os.Getenv("ARCHIVEBOT_NOTIFY")
+				params := slack.PostMessageParameters{}
+				api.PostMessage(onErrorNotify, message, params)
```

Comment:
I'm guessing that this API call will fail if the `ARCHIVEBOT_NOTIFY` environment variable is empty, but that the error will be ignored?  I'd rather be explicit about only attempting the notification if there's a user/channel defined, and it should also log an error if the call fails.